taxi/urls.py

```diff
@@ -16,8 +22,38 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
```

Comment:
```suggestion
    path(
        "manufacturers/create/",
        ManufacturerCreateView.as_view(),
        name="manufacturer-create",
    ),
```
add `/` wherever needed